[ISSUE-509] show wait-fire-event correctly

IvanAprea opened the pull request:
## Checks

- [ ] I have read the [contributing guidelines](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/CONTRIBUTING.md).
- [ ] If some rule is added/updated/removed, I've regenerated the rules list (`npm run generate:rules-list`)
- [ ] If some rule meta info is changed, I've regenerated the plugin shared configs (`npm run generate:configs`)

## Changes

<!-- List the changes you're making with this pull request. -->
Issue link: https://github.com/testing-library/eslint-plugin-testing-library/issues/590

- I changed the waitFireEvent prop from "methodName" to "name"

![image](https://user-images.githubusercontent.com/54630721/169611306-3c31a483-2946-487e-8bf1-f55acb049556.png)
I am showing the error with gitLens

## Context

<!--
If you're fixing an issue with this pull request then use the "Fixes" keyword, like this:
Fixes #123
-->

IvanAprea commented:
@Belco90 :eyes:

Belco90 commented on lib/rules/await-fire-event.ts:
```diff
@@ -29,7 +29,7 @@ export default createTestingLibraryRule<Options, MessageIds>({
-        'Promise returned from `fireEvent.{{ methodName }}` must be handled',
+        'Promise returned from `fireEvent.{{ name }}` must be handled',
       fireEventWrapper:
         'Promise returned from `fireEvent.{{ wrapperName }}` wrapper over fire event method must be handled',
```
I think we have to do the same change here, right?

IvanAprea replied:
I think the same, but change wrapperName to name breaks the tests, i will try to find the problem and fix that :D

Belco90 replied:
Let me know if you need help for this!

Belco90 replied:
You are right! It should be the name of the wrapper 👌

Belco90 reviewed (changes requested):
Thanks for taking care of this!

I think we need to do the change in the other message. Additionally, can you update the tests for this rule so they check that the correponding `name` property is passed to the messages?

IvanAprea commented:
@Belco90 I did another commit with the changes for the messageId: fireEventWrapper. I changed the var to name and fixed the test in order to use the wrapper function name. I saw the rest of tests and i think that they are well. The tests didnt fail before because the var name in the messages was wrong. Please let me know if you have any doubt or if you need that i test anymore

Belco90 commented:
@all-contributors please add @IvanAprea for code and tests
